airbyte-integrations/connectors/source-monday/unit_tests/test_extractor.py

```diff
@@ -34,3 +34,27 @@ def test_extract_records():
     assert len(records) == 1
     assert records[0]["pulse_id"] == 123
     assert records[0]["created_at_int"] == 1636738688
+
+
+def test_extract_records_incremental():
+    # Mock the response
+    response = MagicMock()
+    response_body = {
+        "data": {
+            "boards": [
+                {
+                    "id": 1
+                }
+            ]
+        }
+    }
+
+    response.json.return_value = response_body
+    extractor = MondayIncrementalItemsExtractor(parameters={}, field_path=["data", "ccccc"], config=MagicMock())
+    extractor.field_path_pagination = ["data", "bbbb"]
```

Comment:
Is there a reason why not to provide `field_path_pagination` and `field_path_incremental` during `__init__`?

Furthermore, what does configuring `field_path_pagination` do for this test? It seems like it is unrelated for asserting `records == [{'id': 1}]`

Reply:
this particuar code tests extraction data from response during incremental sync.
Our code tries all possible paths: field_path, field_path_pagination and last field_path_incremental.

So here we test that previous paths failed and the last (field_path_pagination) succeded.